openwebstart/src/main/java/com/openwebstart/controlpanel/SimpleDesktopSettingsPanel.java

```diff
@@ -42,6 +45,7 @@ public SimpleDesktopSettingsPanel(DeploymentConfiguration config) {
             config.setProperty(shortcutComboOptions.getActionCommand(), c.getValue());
         });
 
+        uiLock.update(OwsDefaultsProvider.REMOTE_DEBUG, shortcutComboOptions);
```

Comment:
For me it looks like the `OwsDefaultsProvider.REMOTE_DEBUG` is a copy-paste error.
Can you change this to the correct key please